Core/Dlr/Actions/DynamicMetaObject.cs

```diff
@@ -22,7 +22,7 @@
 using Remotion.TypePipe.Dlr.Dynamic.Utils;
 #if SILVERLIGHT
 using System.Core;
-#else
+#elif FEATURE_REMOTING
```

Comment:
Ah, what I missed: please add Feature_Remoting to the csproj settings for release and debug.